server/project.go

```diff
@@ -603,11 +603,10 @@ func (s *Server) composeProjectByID(ctx context.Context, id int) (*api.Project,
 	return project, nil
 }
 
-func (s *Server) composeProjectRelationship(ctx context.Context, projectRaw *api.ProjectRaw) (*api.Project, error) {
+func (s *Server) composeProjectRelationship(ctx context.Context, raw *api.ProjectRaw) (*api.Project, error) {
 	var err error
```

Comment:
nit: not related to this change, we don't need to pre-define error if possible.

Reply:
But sometimes it seems inevitable, like this one?